src/main/java/com/github/fge/jsonschema/core/processing/CachingProcessor.java

```diff
@@ -53,7 +53,6 @@
     private static final MessageBundle BUNDLE
         = MessageBundles.getBundle(JsonSchemaCoreMessageBundle.class);
 
-    public static final String CACHE_SIZE_PROPERTY_NAME = "com.github.fge.jsonschema.processorCacheSize";
```

Comment:
I think the reason I had done this is that up a layer or two it is kind of difficult for the people using json-schema-validator to poke an override value in.